test/kubemark/start-kubemark.sh

```diff
@@ -408,6 +408,7 @@ current-context: kubemark-context")
   sed -i'' -e "s/{{kubelet_verbosity_level}}/${KUBELET_TEST_LOG_LEVEL}/g" "${RESOURCE_DIRECTORY}/hollow-node.yaml"
   sed -i'' -e "s/{{kubeproxy_verbosity_level}}/${KUBEPROXY_TEST_LOG_LEVEL}/g" "${RESOURCE_DIRECTORY}/hollow-node.yaml"
   sed -i'' -e "s/{{use_real_proxier}}/${USE_REAL_PROXIER}/g" "${RESOURCE_DIRECTORY}/hollow-node.yaml"
+  sed -i'' -e "s/{{hollow_proxy_params}}/${HOLLOW_PROXY_TEST_ARGS}/g" "${RESOURCE_DIRECTORY}/hollow-node.yaml"
```

Comment:
Since you're anyway adding this - maybe we can eliminate USE_REAL_PROXIER and KUBEPROXY_TEST_LOG_LEVEL (and include them as part of this)?

Reply:
OK - though we first need to migrate job configuration. But that's fine. Let me do that.